arangod/V8Server/V8DealerFeature.cpp

```diff
@@ -1607,14 +1608,14 @@ void V8DealerFeature::shutdownContexts() {
 
   // wait for one minute
```

Comment:
Comment not accurate anymore. Can be removed.
```suggestion
```

Reply:
10 * 60 * sleep(10ms) it still true

Reply:
Fixed to `wait no more than one minute`